tailwind.config.js

```diff
@@ -11,12 +11,13 @@ module.exports = {
         unselected: "#A9AFB8",
         button: "#F5F7FA",
         placeholder: "#E0E4E9",
+        bghover: "#F7F9FB",
+        bluetitle: "004284",
```

Comment:
* [nit] `bghover` -> `hover`, so that the resulting background class name becomes `bg-hover` instead of `bg-bghover` (I made this mistake with `border`, which makes it weirdly become `border-border`)
* `bluetitle` got added in #40 as `colors.blue`. You can use "blue" as a color wherever you need it!